srcpkgs/fpc-source/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'fpc-source'
+pkgname=fpc-source
+version=3.0.4
+revision=1
+noarch=yes
+wrksrc="fpc-source"
+create_wrksrc=yes
+build_style=fetch
+short_desc="Source code for the Freepascal compiler and libs"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="GPL-2.0-or-later, modified LGPL-2.0-or-later"
+homepage="https://www.freepascal.org/"
+distfiles="$SOURCEFORGE_SITE/freepascal/Source/$version/fpc-$version.source.tar.gz"
+checksum=69b3b7667b72b6759cf27226df5eb54112ce3515ff5efb79d95ac14bac742845
+nostrip=yes
+do_build() {
+	tar -xf "$wrksrc"/fpc*.tar.gz
```

Comment:
It's same as default. Remove `build_style=` and `do_build`.

Reply:
Fixed.